JuiceMaker/JuiceMaker/Model/Juice.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+enum Juice {
+    case strawberryJuice
+    case bananaJuice
+    case kiwiJuice
+    case pineappleJuice
+    case strawberryBananaJuice
+    case mangoJuice
+    case mangoKiwiJuice
+    
+    var recipe: [Fruit : Int] {
+        switch self {
+        case .strawberryJuice:
+            return [.strawberry : 16]
+        case .bananaJuice:
+            return [.banana : 2]
+        case .kiwiJuice:
+            return [.kiwi : 3]
+        case .pineappleJuice:
+            return [.pineapple : 2]
+        case .strawberryBananaJuice:
+            return [.strawberry : 10, .banana : 1]
+        case .mangoJuice:
+            return [.mango : 3]
+        case .mangoKiwiJuice:
+            return [.mango : 2, .kiwi : 1]
+        }
+    }
```

Comment:
```suggestion
    var recipe: [Fruit: Int] {
        switch self {
        case .strawberryJuice:
            return [.strawberry: 16]
        case .bananaJuice:
            return [.banana: 2]
        case .kiwiJuice:
            return [.kiwi: 3]
        case .pineappleJuice:
            return [.pineapple: 2]
        case .strawberryBananaJuice:
            return [.strawberry: 10, .banana: 1]
        case .mangoJuice:
            return [.mango: 3]
        case .mangoKiwiJuice:
            return [.mango: 2, .kiwi: 1]
        }
    }
```
요기도 컨벤션 맞춰주면 좋을 것 같아요!